src/module.c

```diff
@@ -190,6 +197,15 @@ int module_preload(const char *module)
 	pl_set_str(&path, ".");
 	pl_set_str(&name, module);
 
+	char file[FS_PATH_MAX];
+	if (re_snprintf(file, sizeof(file), "%r/%r", &path, &name) < 0) {
```

Comment:
Perhaps better to use re_sdprint to avoid a large buffer on the stack ?

Reply:
Will do.

Reply:
Addressed with commit `8e5b0bdf`.